share/spice/rx_info/rx_info.js

```diff
@@ -19,40 +19,45 @@
         return concatted;
     }
 
+    // Use matchedTerms object to build secondaryText string
+    function buildSecondaryText(matchedTerms) {
+        var secondaryText = "";
+
+        // Loop through matchedTerms object for terms and values
+        for (var key in matchedTerms) {
+           if (matchedTerms.hasOwnProperty(key)) {
+              secondaryText += key + "=" + matchedTerms[key] + ", ";
+           }
+        }
+
+        // If terms were found, make secondaryText human readable
+        if (secondaryText.length > 0) {
+            secondaryText = "Matched: " + secondaryText.substr(0, secondaryText.length - 2) + ".";
+        }
+
+        return secondaryText;
+    }
+
     env.ddg_spice_rx_info = function(api_result){
 
         if (!api_result || api_result.error || !api_result.replyStatus || !api_result.replyStatus.totalImageCount || api_result.replyStatus.totalImageCount < 1) {
             return Spice.failed('rx_info');
         }
         
-        var script = $('[src*="/js/spice/rx_info/"]')[0],
-            source = $(script).attr("src"),
-            triggerWordMatch = parseInt(source.match(/rx_info\/[^\/]+\/(\d)/)[1]),
-            relCheck;
-        
+        var sourceName = "DailyMed",
+            sourceUrl = "http://dailymed.nlm.nih.gov/",
```

Comment:
@moollaza Assuming you are referring to the `sourceName` and `sourceUrl`, unfortunately at this time there is no single page that could be linked that would reflect the users search with any degree of specificity.  It may be less confusing to the end user to just remove this text / link for now?